internal/xdscache/v3/route.go

```diff
@@ -143,6 +143,14 @@ func (v *routeVisitor) onVirtualHost(vh *dag.VirtualHost) {
 				Action: envoy_v3.UpgradeHTTPS(),
 			}
 		}
+
+		if route.DirectResponse != nil {
+			return &envoy_route_v3.Route{
+				Match:  envoy_v3.RouteMatch(route),
+				Action: envoy_v3.RouteDirectResponse(route.DirectResponse),
+			}
+		}
```

Comment:
Just wondering if we should still apply request/response headers, rate limit policies to the route even if it's a direct response. I guess it's kind of a moot point right now since those things aren't supported yet for Gateway, and we can revisit if/when we support direct response for HTTPProxy.

Reply:
I think it's a great point that we should keep in mind when Gateway gets that support and also to add to HTTPProxy when we do that.